packages/core/src/components/tooltip/tooltip.tsx

```diff
@@ -21,17 +20,16 @@ export interface ITooltipProps extends IProps, IIntentProps {
     content: JSX.Element | string;
 
     /**
-     * Constraints for the underlying Tether instance.
-     * See http://github.hubspot.com/tether/#constraints
-     * @deprecated since v1.12.0; use `tetherOptions` instead.
+     * Initial opened state when uncontrolled.
+     * @default false
```

Comment:
remove this, it defaults to `undefined`

Reply:
Doesn't it default explicitly to `false` below?

https://github.com/palantir/blueprint/pull/1891/files/08312ffb58f969cfa74dc3738982393d142cdc07#diff-0f4e5f29db46149ff7a16aa9869d0ff9R130